infrastructure/swagger/05_paths.yaml

```diff
@@ -288,3 +288,33 @@ paths:
       security:
         - ${authoriser_name}: []
         - app-level0: []
+
```

Comment:
As this will go in after the path refactoring, should we make it match that ticket PI-525?